src/client/Health/NodeView.jsx

```diff
@@ -25,37 +39,258 @@ const height = 500
+        value: { dataSourceId, protocol, enabled: false },
+      })
+    } else {
+      const error = new Error('data source already exists')
+      setAlert({ text: error.message, type: 'danger' })
+      throw error
+    }
+  }
+
+  const handleDeleteSouth = (position) => {
+    dispatchNewConfig({ type: 'deleteRow', name: `south.dataSources.${sortableDataSources[position].index}` })
+  }
+
+  const handleDuplicateSouth = (position) => {
+    const dataSource = dataSources[sortableDataSources[position].index]
+    const newName = `${dataSource.dataSourceId} copy`
+    const countCopies = dataSources.filter((e) => e.dataSourceId.startsWith(newName)).length
+    dispatchNewConfig({
+      type: 'addRow',
+      name: 'south.dataSources',
+      value: {
+        ...dataSource,
+        dataSourceId: `${newName}${countCopies > 0 ? countCopies + 1 : ''}`,
+        enabled: false,
+      },
+    })
+  }
+
+  const handleStatusSouth = (position) => {
+    const dataSource = sortableDataSources[position]
+    const pathname = `/south/${dataSource.dataSourceId}/live`
+    history.push({ pathname })
+  }
+
+  const handleEditSouth = (position) => {
+    const dataSource = sortableDataSources[position]
+    const pathname = `/south/${dataSource.dataSourceId}`
+    history.push({ pathname })
+  }
+
+  const handleDataSourceIdChanged = (dataSourceIndex, newDataSourceId) => {
+    setRenamingConnector(null)
+    dispatchNewConfig({
+      type: 'update',
+      name: `south.dataSources.${dataSourceIndex}.dataSourceId`,
+      value: newDataSourceId,
+    })
+  }
+  // End Tools *******************************
 
   const northNodes = applications?.map((application, index) => (
+
     {
       id: application.applicationId, // unique id of node
       type: 'output', // output node
       targetPosition: 'bottom', // handle is at the bottom
       style: {
-        background: colors.background[application.enabled ? 'enabled' : 'disabled'],
-        border: colors.border[application.enabled ? 'enabled' : 'disabled'],
-        width: '140px',
+        background: colors.background[application.enabled ? 'disabled' : 'disabled'],
+        border: colors.border[application.enabled ? 'disabled' : 'disabled'],
+        width: '170px',
+        height: '25%',
       },
       data: {
         label: (
-          <div id={`north-${index}`} className={`oi-box tight text-${application.enabled ? 'success' : 'muted'}`}>
-            <Link to={`/north/${application.applicationId}`}>
+          <div className="box-container">
+            <div className="icon-container">
+              <div className="icon-left">
+                <img src={`${ApiSchemas[application.api].image}`} alt="logo" height="20px" />
+              </div>
+              <div className="icon-center flex-grow">
+                {`${application.api}`}
+              </div>
+              <div className={`icon-right ${application.enabled ? 'icon-status-enabled' : 'icon-status-disabled'}`} />
```

Comment:
You can condense the code like that : `icon-right icon-status-${application.enabled ? 'enabled' : 'disabled'}`